rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/views/compact/CompactFlowPanel.java

```diff
@@ -69,7 +69,12 @@ public CompactFlowPanel() {
         this.setWidth("100%");
         // removes the vertical space between lines
         this.getElement().getStyle().setProperty("lineHeight", "0");
-        nodeRemover = new NodeRemover(this);
+        this.nodeRemover = new NodeRemover(this);
+    }
+
+    // for testing purpose
```

Comment:
could you use @VisibleForTesting, it is more common

Reply:
did not know about it, i will use it. thanks